phpunit.xml.dist

```diff
@@ -20,10 +18,15 @@
         <env name="MONGODB_DATABASE" value="unittest"/>
         <env name="SQLITE_DATABASE" value=":memory:"/>
         <env name="QUEUE_CONNECTION" value="database"/>
+
+        <ini name="memory_limit" value="-1"/>
```

Comment:
While running the tests locally in Docker I ran into the issue that there wasn't enough memory. Disabling the limit here, seems like the most logical place